pkgs/applications/audio/mpg123/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv
 , fetchurl, alsaLib
-, buildPlatform, hostPlatform
+, hostPlatform
```

Comment:
@Ericson2314: Speaking of stdenv + build/host/targetPlatform... have you considered moving them into stdenv? IMHO, that's a natural place for them. Also, then package files don't have to explicitly add them as inputs.